codeship-services.yml

```diff
@@ -7,7 +7,7 @@ db:
         MYSQL_PASSWORD: silauth
 
 web:
-    image: silintl/ssp-base:latest
+    image: silintl/ssp-base:develop
```

Comment:
Is there benefit in having Codeship use `latest` (to ensure what we're deploying works with our latest production release of ssp-base) and just have the local docker-compose.yml file use `develop`?

Reply:
Fair enough. However, I would like to get away from `latest` since one can never be quite sure what it means (and usually has nothing to do with latest in a timeline sense). But in this case, there doesn't seem to be a `master` tag. Actually, I might go ahead and create a `master` tag and change this reference. Would that be ok with you?

Reply:
From a Docker perspective, `latest` is the `master` tag. The Docker image tagged as `latest` is the most recent image that was build from the `master` branch in the repo.

Reply:
I'll defer to you, then, since that's true at least for this image. I will change this back to `latest`.